pydantic/fields.py

```diff
@@ -315,11 +323,13 @@ def Field(
     default: Any = Undefined,
     *,
     default_factory: typing.Callable[[], Any] | None = None,
-    alias: str | None = None,
+    alias: str | list[str | int] | list[list[str | int]] | None = None,
```

Comment:
```suggestion
    alias: str | None = None,
```

Reply:
For now, I think we should disallow the other kinds of alias to be passed into this function as the `alias` kwarg (as I think it will break type-checking anyway without a plugin or similar), and just allow the more complex variants in `validation_alias`.